packages/ipfs/test/core/create-node.spec.js

```diff
@@ -250,4 +250,28 @@ describe('create node', function () {
 
     await node.stop()
   })
+
+  it('should error when receiving websocket-star swarm addresses', async () => {
+    const node = await IPFS.create({
+      repo: tempRepo,
+      init: { bits: 512 },
+      start: false,
+      config: {
+        Addresses: {
+          Swarm: ['/ip4/127.0.0.1/tcp/13579/wss/p2p-websocket-star']
+        },
+        Bootstrap: []
+      },
+      preload: { enabled: false }
+    })
+
+    try {
+      await node.start()
+    } catch (err) {
+      expect(err).to.exist()
```

Comment:
Please assert the correct error was thrown.

Better yet, do something like

```js
await expect(node.start()).to.eventually.be.rejectedWith(/regex to use on error message/)
```

or

```js
await expect(node.start()).to.eventually.be.rejected().with.property('code', 'ERR_SOME_CODE')
```

then you don't need the try/catch and the following throw.